integration_with_openshift_container_platform/_topics/ocp-metrics.md

```diff
@@ -51,16 +51,18 @@ To create these accounts:
     **metrics-deployer** service account. Configure the
     **metrics-deployer** account to have **edit** permissions in the
     **openshift-infra** project:
-
-        $ oadm policy add-role-to-user \
-            edit system:serviceaccount:openshift-infra:metrics-deployer
-
+    ````
+       $ oc adm policy add-role-to-user \
+    edit system:serviceaccount:openshift-infra:metrics-deployer
+    ```
 6.  The **heapster** account will be automatically created in [Deploying the Metrics Components](#ocp-metrics-deploy). However, pre-emptively
     grant it **cluster-reader** permission to the **openshift-infra**
     project:
 
-        $ oadm policy add-cluster-role-to-user \
-            cluster-reader system:serviceaccount:openshift-infra:heapster
+    ```
+       oc adm policy add-cluster-role-to-user \
+    cluster-reader system:serviceaccount:openshift-infra:heapster
+    ```
```

Comment:
why was the alignment changed from 

```
oc adm policy add-cluster-role-to-user \
    cluster-reader system:serviceaccount:openshift-infra:heapster
```

to

```
   oc adm policy add-cluster-role-to-user \
cluster-reader system:serviceaccount:openshift-infra:heapster
```

Reply:
I have fixed the alignment.